src/Reflow/Parser.hs

```diff
@@ -62,8 +59,10 @@ codeBlock = do
 singleLine :: Parser Text
 singleLine = pack <$> manyTill anyChar (try eol)
 
-quoteChar :: Parser Text
-quoteChar = pack <$> (string ">")
+quotePrefix :: Parser Text
+quotePrefix = fmap pack $ mappend
+    <$> (string ">")
+    <*> (many $ char ' ')
```

Comment:
I believe `spaces` includes newlines, which I _don't_ want to match here. I only want to strip literal space characters.